src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
+
+  if (days >= 7) {
+    sale = 50;
+  } else if (days >= 3) {
+    sale = 20;
```

Comment:
Similarly to the previous comment, avoid using literals for the discount values and minimum days. Define constants like `const SHORT_TERM_DISCOUNT = 20;` and `const SHORT_TERM_THRESHOLD = 3;`.